src/App.tsx

```diff
@@ -1,23 +1,19 @@
-import React, { ChangeEvent, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { Container } from 'react-bootstrap';
 import { TodoItem } from './types';
 import { AddItemForm, TodoList } from './components';
 
 function App() {
-  const [item, setItem] = useState('');
   const [list, setList] = useState<TodoItem[]>([]);
 
-  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
-    if (e.target.name === 'input') {
-      setItem(e.target.value);
-    }
-  };
+  useEffect(() => {
+    localStorage.setItem("list", JSON.stringify(list));
```

Comment:
mb add move descriptive name to this key?

Reply:
I think in the future you can have problems because you have such generic name...